### ISSUE TITLE
embedding function

### ISSUE THREAD

**Issue by huili03855** [BangLiu/QANet-PyTorch#2]
Why do we need "ch_emb = ch_emb.squeeze()" in the Embedding class in QANet_andy.py. When I test with batch_size = 1, this code will made the dim of ch_emb lower than wd_emb, which leads an error when we execute "emb = torch.cat([ch_emb, wd_emb], dim=1)"

**Comment by BangLiu** [comment]
@huili03855 Yeah you are right, I tested it, we should remove that line.
